### ISSUE TITLE
input never an integer in custom_network function - network_creator 

### ISSUE THREAD

**Issue by A-acuto** [dstl/YAWNING-TITAN#14]
Hi, 

I think there is a piece missing in the function custom_network(), from network_creator code (helper functions).

The input number of nodes is checked if it is an integer, however every input from user is read as string (or at least in a Windows environment). So it is not possible to create a custom network.

An easy fix could be:
**Actual code**
```
def custom_network() -> Union[Tuple[np.array, dict], None]:
    size = input("How many nodes in the network? ")
    if type(size) != int:
        print("Error in input - NOT INT")
        return None
    else:
        #do something
```
**FIX**
```
def custom_network() -> Union[Tuple[np.array, dict], None]:
    size = input("How many nodes in the network? ")
    if type(int(size)) != int:
        print("Error in input - NOT INT")
        return None
    else:
        #do something
```

I think in this way if the input is a float or a text char it will not work, however it will go smoothly for integers.
I hope this helps, 
Alberto

**Comment by jamesshort1** [comment]
Added as a new IDT Jira issue (AIDT-96)

**Comment by ChrisMcCarthyDev** [comment]
Hi @A-acuto. Thank you for raising this. I can confirm that a fix for this will be included in the imminent v1.0.0 release.
